src/utils.ts

```diff
@@ -12,6 +12,8 @@ import {
     ALPHA_MAINNET_INTERNALLY,
     ALPHA_TESTNET,
     ALPHA_TESTNET_INTERNALLY,
+    ALPHA_TESTNET_TWO,
```

Comment:
Why use TWO instead of 2?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/258/commits/2c7386c5c9ff8f72caefd39c824a0b9101136789